pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -18,11 +18,13 @@
 """
 
 import datetime as dt
+import gzip
```

Comment:
Is this in the standard library?

Reply:
It is included into python core as far as I understand https://docs.python.org/3/library/gzip.html